File: IssueDetails.java
import org.json.JSONArray;
import org.json.JSONObject;

import com.google.gson.JsonArray;
import com.google.gson.JsonObject;

public class IssueDetails {
	public JSONObject getData(String id,JsonArray arr) {
		JSONObject obj = new JSONObject();	
		
		for(int i=0; i< arr.size();i++) {
			JsonObject jsonObject1 = arr.get(i).getAsJsonObject();
			if(jsonObject1.get("number").getAsString().equalsIgnoreCase(id)) {
			obj.put("issue_id",jsonObject1.get("number").getAsString());
			obj.put("id",jsonObject1.get("id").getAsString());
			obj.put("created",jsonObject1.get("created_at").getAsString());
			obj.put("updated",jsonObject1.get("updated_at").getAsString());
			obj.put("author",jsonObject1.get("author_association").getAsString());
			obj.put("body",jsonObject1.get("body").getAsString());
			obj.put("url",jsonObject1.get("html_url").getAsString());
			System.out.println(obj);
			return obj;
			}
		}
		return obj;
	}
	
  
}

File: issuedetails.js

var details= angular.module("details", ["ngMessages","ngMaterial","ngAria"]);

details.controller("detailsController", [ '$scope', '$http','$window','$mdDialog', function($scope, $http,$window,$mdDialog) {
     $scope.addFields = function(ev) {
     $http({
         url : 'HomeService',
         method : "POST",
         params : {
             'issueId' : id,
             'mode':'GETDATA'
         }
     }).then(function(response,$window) {
         console.log(response.data);
         $scope.object = response.data;
         var container = document.getElementById("container");
         while (container.hasChildNodes()) {
             container.removeChild(container.lastChild);
         }
             var para = document.createElement("p");
             var input = document.createElement("textarea");
             input.maxLength ="400";
             input.rows ="30";
              input.setAttribute('ng-model',response.data.issue_id);
              input.value = "Issue ID  :: " +response.data.id+"\r\n"+"Issue Number ::"+response.data.issue_id+" "+"\r\n"+"Created on ::"+response.data.created+" "+"\r\n"
              + "Updated On ::: " +response.data.updated+"\r\n" + "Author association :: " +response.data.author+" "+"\r\n" 
              + "Body ::: " +response.data.body+"\r\n" + "Url : " +response.data.url;
            
            para.appendChild(input);
            container.appendChild(para);
        })
    }     ;
    

     
}]);
